src/vs/workbench/browser/actions/layoutActions.ts

```diff
@@ -650,12 +650,15 @@ registerAction2(ShowEditorActionsAction);
 
 // --- Editor Actions Position Submenu in View Appearance Menu
 
-MenuRegistry.appendMenuItem(MenuId.MenubarAppearanceMenu, {
-	submenu: MenuId.EditorActionsPositionSubmenu,
-	title: localize('editorActionsPosition', "Editor Actions Position"),
-	group: '3_workbench_layout_move',
-	order: 11
-});
+// --- Start Positron ---
+// Do not append the Editor Actions Position submenu. It has been replaced by the Editor Action Bar.
+// MenuRegistry.appendMenuItem(MenuId.MenubarAppearanceMenu, {
```

Comment:
To reduce merge conflicts, could you set this off with `/*` so that the text upstream can change w/o generating a conflict?